heartbeat/monitors/logger/logger.go

```diff
@@ -130,6 +135,29 @@ func extractRunInfo(event *beat.Event) (*MonitorRunInfo, error) {
 	return &monitor, nil
 }
 
+func extractNetworkInfo(event *beat.Event, monitorType string) NetworkInfo {
+	// Only relevant for lightweight monitors
+	if monitorType == "browser" {
+		return nil
+	}
+
+	fields := []string{
+		"resolve.rtt.us", "tls.rtt.handshake.us",
```

Comment:
@andrewvc Do you see any problem with also logging `resolve.ip` address? Makes it easier for debugging timeouts.